public/index.html

```diff
@@ -49,7 +49,7 @@
 
         <!-- Load custom CSS -->
         <link rel="stylesheet" href="css/main.css" />
-        <link rel="stylesheet" href="css/reading.css" />
```

Comment:
I think you should be adding a new link tag and not remove the reading.css files

Reply:
+1

Reply:
Thanks for catching this too